### PR TITLE
update README, remove stale comments

### PR DESCRIPTION
**Pull request by master-hax**
updates "Webex Teams" to "Webex", removes stale comment that is now misleading

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -1008,9 +1008,6 @@ Bot.prototype.sayWithLocalFile = function (message, filename) {
 
 /**
  * Send a threaded message reply
- * 
- * NOTE:  Posting a threaded message response via API is currently a Webex EFT feature.  This method WILL FAIL
- * if your application identity is not configured for EFT access
```
Comment (master-hax) [inline lib/bot.js:1013]: stale

File: README.md [hunk 2]
```diff
@@ -1082,9 +1082,6 @@ framework.hears('/file', function(bot, trigger) {
 ### bot.reply(replyTo, message, [format]) ⇒ <code>Promise.&lt;Message&gt;</code>
 Send a threaded message reply
 
-NOTE:  Posting a threaded message response via API is currently a Webex EFT feature.  This method WILL FAIL
-if your application identity is not configured for EFT access
-
```
Comment (master-hax) [inline README.md:1087]: stale

File: lib/bot.js [hunk 3]
```diff
@@ -1009,9 +1009,6 @@ Bot.prototype.sayWithLocalFile = function (message, filename) {
 /**
  * Send a threaded message reply
  * 
- * NOTE:  Posting a threaded message response via API is currently a Webex EFT feature.  This method WILL FAIL
- * if your application identity is not configured for EFT access
- *
```
Comment (master-hax) [inline lib/bot.js:1014]: stale

File: README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
 # webex-node-bot-framework
 
-### Webex Teams Bot Framework for Node JS
+### Node JS framework for Cisco Webex
```
[the remark this answers is not in the document]
Comment (master-hax) [reply README.md:3]: changed to "Node JS Bot Framework for Cisco Webex".

my goal was simply to remove "Webex Teams" from the beginning as it makes it look like this repo is a Cisco product at first glance when it is actually an open-source project *for* a Cisco product.

File: README.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+This project is inspired by, and provides an alternate implementation of, the awesome [node-flint](https://github.com/flint-bot/flint/) framework by [Nick Marus](https://github.com/nmarus).  The framework makes it easy to quickly develop a Webex messaging bot, abstracting away some of the complexity of Webex For Developers interfaces, such as registering for events and calling REST APIs. A bot developer can use the framework to focus primarily on how the bot will interact with users in Webex Teams, by writing "handlers" for various message or membership events in spaces where the bot has been added.
```
[the remark this answers is not in the document]
Comment (master-hax) [reply README.md]: fixed this & commited - then installed deps & ran `docs/build.sh` & committed updated README.

**Review by jpjpjp — APPROVED** [review]
Looks good.  Thanks for the contribution!

**Pull request merged**